### ISSUE TITLE
Fails to install with 4.04.0

### ISSUE THREAD

**Issue by andreas** [mirage/ocaml-uri#92]
When installing `ocaml-uri` under 4.04.0 with opam, it fails with the following message:

```
Error: broken invariant in parsetree: Let with no bindings.
```

It sounds like this error is related to:

- https://github.com/ocaml/ocaml/pull/392
- https://caml.inria.fr/mantis/view.php?id=7111

I would love to help fix this, but I haven't been able to find any good resources on what "let with no bindings" means.

This is possibly a duplicate of https://github.com/mirage/ocaml-uri/issues/91, but that issue contains very little information, so I'm not sure.

**Comment by hcarty** [comment]
#91 is/was due to being blocked by a ppx dependency which wasn't available during the 4.04.0 beta period.  I haven't tested since.

**Comment by dsheets** [comment]
This looks like an issue with `ppx_sexp_conv` 113.33.00+4.03 which was fixed in 113.33.01+4.03. I've updated the package metadata to avoid the buggy version. Could you please try the newer version and check if that solves the problem?

**Comment by andreas** [comment]
It works now. Thanks!
